packages/@okta/vuepress-site/code/ios/index.md

```diff
@@ -49,12 +49,18 @@ The Okta AppAuth iOS SDK is a wrapper around the AppAuth-iOS SDK for communicati
 		<i class='fa fa-github'></i>
 		<a href="https://github.com/okta/okta-ios-jwt">
 			<span>Okta JWT Verifier for iOS</span>
+      <br/>
+      <br/>
```

Comment:
I missed this earlier, but I morally object to <br/> tags used like this.  Something to follow up on in a later PR :)

Reply:
Noted 😄